vite.config.ts

```diff
@@ -12,8 +12,26 @@ export default defineConfig({
   build: {
     outDir: "dist",
     sourcemap: true,
+    rollupOptions: {
+      external: [
+        "@resvg/resvg-js",
+      ],
+    },
   },
   define: {
     global: "globalThis",
   },
+  resolve: {
+    alias: {
+      "@resvg/resvg-js": "@resvg/resvg-wasm",
+      buffer: "buffer/",
+    },
+  },
+  optimizeDeps: {
+    esbuildOptions: {
+      define: {
+        global: "globalThis",
+      },
+    },
+  },
```

Comment:
Can you make sure youre only introducing necessary options

Reply:
> Can you make sure youre only introducing necessary options

my bad, these are left over from debugging. Removed